src/components/MobileNav/NavItem.tsx

```diff
@@ -5,15 +5,15 @@ import ReactSVG from "react-svg";
 import { NavLink } from "..";
 import { MainMenuSubItem } from "../MainMenu/types/MainMenuSubItem";
 
-const subcategoriesIcon = require("../../images/subcategories.svg");
+import subcategoriesImg from "../../images/subcategories.svg";
 
-interface NavItem extends MainMenuSubItem {
-  children?: NavItem[];
+export interface NavItemInterface extends MainMenuSubItem {
```

Comment:
It would be better if you use `INavItem` instead of `NavItemInterface` if you prefer to denote that it's the name of the interface.